model/GlobalConfig.php

```diff
@@ -88,31 +90,89 @@ private function initializeConfig()
                 $this->parseConfig($this->filePath);
             }
 
-            $configResources = $this->graph->allOfType("skosmos:Configuration");
-            if (is_null($configResources) || !is_array($configResources) || count($configResources) !== 1) {
-                throw new Exception("config.ttl must have exactly one skosmos:Configuration");
-            }
-
-            $this->resource = $configResources[0];
             $this->initializeNamespaces();
         } catch (Exception $e) {
             echo "Error: " . $e->getMessage();
         }      
     }
 
     /**
-     * Parses configuration from the config.ttl file
-     * @param string $filename path to config.ttl file
+     * Ensure there is exactely one skosmos:Configuration and return it.
+     */
+    private function configResource($graph, $source) {
+        $configResources = $graph->allOfType("skosmos:Configuration");
+        if (is_null($configResources) || !is_array($configResources) || count($configResources) !== 1) {
+            throw new Exception("$source must have exactly one skosmos:Configuration");
+        }
+        return $configResources[0];
+    }
+
+    /**
+     * Retrieves and parses configuration with optional inclusion.
+     * @param string file or URL of configuration in Turtle syntax.
+     * @param int inclusion depth (0 for root configuration)
      * @throws \EasyRdf\Exception
      */
-    private function parseConfig($filename)
-    {
-        $this->graph = new EasyRdf\Graph();
+    private function parseConfig($location, $depth=0) {
+        if (str_starts_with($location, "http://") || str_starts_with($location, "https://")) {
+            $ch = curl_init($location);
+            curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+            curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-type: text/turtle'));
```

Comment:
Shouldn't this be `Accept: text/turtle` ? Content-type is a response header, not a request header.